sphinx/source/docs/user_guide.rst

```diff
@@ -63,6 +63,9 @@ topic:
 :ref:`userguide_geo`
     Working with geographical data - Google Maps, GeoJSON, Tile Rendering.
 
+:ref:`userguide_graphs`
+    Create network graph visualizations with configurable node and edge interactions.
+
```

Comment:
I think these should be moved higher. I think the order should be:

* plotting basic glyphs
* handling catgorical
* network/graph
* geo